scipy/fftpack/tests/test_basic.py

```diff
@@ -199,6 +199,28 @@ def test_notice(self):
         pass
 
 
+class TestFloat16FFT(TestCase):
+    def setUp(self):
+        self.cdt = np.complex64
+        self.rdt = np.float16
```

Comment:
I did originally, but changed it later and didn't remove the setUp.  will remove it